### PR TITLE
Use n helper to install Node.js

### PR DESCRIPTION
**Pull request by JimboJoe**
I'm just opening this PR to tell you that I worked a little further what @frju365 had started with the n helper.

What's missing now:
- [x] upgrade
- [x] backup/restore
- [ ] completely happy package_check :wink:

### CODE REVIEW

**Comment by JimboJoe** [comment]
You also got an answer to this [upstream issue](https://github.com/xwiki-labs/cryptpad/issues/147).

File: scripts/install [hunk 1]
```diff
@@ -95,58 +116,49 @@ SETUP_SOURCE	# Télécharge la source, décompresse et copie dans $final_path
...
 script_dir="$PWD"
 pushd "$final_path"
 sudo chown -R $app: $final_path
-sudo su $app -c "cd $final_path && npm install"
-sudo npm install -g bower
-sudo su $app -c "cd $final_path && bower install && bower install chainpad#^0.4.0"
+sudo_path npm install
+sudo_path npm install -g bower
+sudo su -l $app -s /bin/bash -c "cd $final_path && env PATH=$PATH bower install"
+# && bower install chainpad#^0.4.0
```
Comment (JimboJoe) [inline scripts/install]: I also commented this out, but maybe you'd figured out it was necessary... ?

File: scripts/install [hunk 2]
```diff
@@ -66,19 +66,40 @@ port=$(ynh_find_port 3000)	# Cherche un port libre.
 ynh_app_setting_set $app port $port
 
 #=================================================
-# INSTALL DEPENDENCIES
+# INSTALL NODEJS
 #=================================================
+ynh_install_nodejs 8.0.0
```
Comment (JimboJoe) [inline scripts/install]: I didn't find where the required version was specified in cryptpad doc; any URL?

File: scripts/install (hunk 1 above)
Comment (magikcypress) [inline scripts/install]: I don't know ! :)

File: scripts/install (hunk 2 above)
Comment (frju365) [inline scripts/install]: They say the latest version. I just choose 8.0.0 but indeed, it's better to choose 8.2.1

**Comment by frju365** [comment]
We need to solve the conflict too.

**Comment by frju365** [comment]
But then, I think we can merge and test after, because we won't break any installation : it already doesn't work ;)

**Pull request merged**